test/common/buffer/buffer_fuzz.cc

```diff
@@ -165,6 +165,15 @@ class StringBuffer : public Buffer::Instance {
 
   std::string toString() const override { return std::string(data_.data() + start_, size_); }
 
+  void setWatermarks(uint32_t) override {
+    // Not implemented.
+    ASSERT(false);
+  }
+  uint32_t highWatermark() const override { return 0; }
+  // Returns true if the high watermark callbacks have been called more recently
+  // than the low watermark callbacks.
```

Comment:
optional: can remove this comment given it's an override and commneted in the base class.

Reply:
Done.